src/ert/scheduler/lsf_driver.py

```diff
@@ -266,7 +266,7 @@ async def submit(
 
     async def kill(self, iens: int) -> None:
         if iens not in self._iens2jobid:
-            logger.error(f"LSF kill failed due to missing jobid for realization {iens}")
+            logger.debug(f"LSF kill failed due to missing jobid for realization {iens}")
```

Comment:
I think this should be kept an error, because it should not happen (lets fix the bug that triggers this too often instead of downgrading to DEBUG)